Python/DataAugmentationUtilsPackage/DataAugmentationUtils/__init__.py

```diff
@@ -11,7 +11,7 @@
 - aug_type = type of augmentaion to run, 1 is for aug based only on particles, 2 bases aug on images and particles
 - sampler_type = type of distribution to represent embedded data with for sampling
 '''
-def runDataAugmentation(out_dir, img_list, point_list, num_samples, num_dim=0, sampler_type="KDE", mixture_num=0):
+def runDataAugmentation(out_dir, img_list, point_list, num_samples, num_dim=0, sampler_type="kde", mixture_num=0):
```

Comment:
- Isn't num_dim the same as num_pca_modes? I would be consistent in argument naming. 
- I would also expose percent_variabililty as input argument here.
- mixture_num -> num_mixture_components

Reply:
I am not sure how to allow either num_dim or percent_variabilty as an argument... The use would have to specify one and set the other to a value like -1 or something to indicate not to use it. Do you have an idea of how to do this?

Reply:
percent_variability can be set as 0.95 by default. num_pca_modes to 0 so the default behavior would be finding the number of modes that capture 95% of the data variability.